recipes/concurrencpp/all/patches/directory-name.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -99,8 +99,8 @@ include(CMakePackageConfigHelpers)
+ include(CMakePackageConfigHelpers)
+ include(GNUInstallDirs)
+ 
+-set(concurrencpp_directory "concurrencpp-${PROJECT_VERSION}")
+-set(concurrencpp_include_directory "${CMAKE_INSTALL_INCLUDEDIR}/${concurrencpp_directory}")
++set(concurrencpp_directory "concurrencpp")
++set(concurrencpp_include_directory "${CMAKE_INSTALL_INCLUDEDIR}")
```

Comment:
This ensures headers are installed into `include/concurrencpp`. Without this patch, they are installed into `include/concurrencpp-0.1.4/concurrencpp`.